source/index.md

```diff
@@ -52,162 +61,1289 @@ The Horus API does a division of authentication according to the type of user, W
+        }, 
+        "data":null
+      }, 
+      "client-telephones":
+      {
+        "links":
+        {
+          "self":"http://localhost:3000/api/v1/manager/clients/fec0f2a6-ac7c-4e51-94e3-9ba222b0e467/relationships/client-telephones", 
+          "related":"http://localhost:3000/api/v1/manager/clients/fec0f2a6-ac7c-4e51-94e3-9ba222b0e467/client-telephones"
+        }
+      }
+    }
+  }
+}
+```
+
+### Update
+
+You also can update a resource using a command line, with the id of the resource, or accessing a route, both cases are described below. In this exeample we use the manager.
+
+Route: `PATCH "/api/v1/manager/clients/:id"`
+
+> Update Resource
+
+```ruby
+horus-cli update clients "id_client" '"corporate-name":"Corporate_01", "trade-name":"Trade_01", "key-name":"key", "email":"client_01@example.com"'
+
+# Return
+
+{
+  "id":"690c7235-59b3-5a4e-9c04-e1aa98cc6e38", 
+  "type":"clients", 
+  "corporate-name":"Corporate_01", 
+  "trade-name":"Trade_01", 
+  "key-name":"key", 
+  "email":"client_01@example.com"
+}
+```
+
+```shell
+# The access_token is obtained in the login.
+
+curl --header "Content-Type: application/vnd.api+json" -H "Authorization: Bearer 'access_token'" -X PATCH -d 
+'{
+  "data":
+  {
+    "id":"client_id", 
+    "type":"clients", 
+    "attributes":
+    { 
+      "corporate-name":"Corporate_03", 
+      "trade-name":"Trade_03", 
+      "email":"client_03@example.com"
+    }
+  }
+}' 
+http://192.168.2.15:3000/api/v1/manager/clients/client_id
+
+# Return
+
+{
+  "data":
+  {
+    "id":"918ba9c6-bde4-543a-a280-afbc1bf698ad", 
+    "type":"clients", 
+    "links":
+    {
+      "self":"http://192.168.2.15:3000/api/v1/manager/clients/918ba9c6-bde4-543a-a280-afbc1bf698ad"
+    },
+    "attributes":
+    {
+      "corporate-name":"Corporate_03",
+      "trade-name":"Trade_03",
+      "key-name":"smart",
+      "email":"client_03@example.com"
+    },
+    "relationships":
+    {
+      "client-address":
+      {
+        "links":
+        {
+          "self":"http://192.168.2.15:3000/api/v1/manager/clients/918ba9c6-bde4-543a-a280-afbc1bf698ad/relationships/client-address", 
+          "related":"http://192.168.2.15:3000/api/v1/manager/clients/918ba9c6-bde4-543a-a280-afbc1bf698ad/client-address"
+        },
+        "data":
+        {
+          "type":"client-addresses","id":"918ba9c6-bde4-543a-a280-afbc1bf698ad"
+        }
+      },
+      "client-telephones":
+      {
+        "links":
+        {
+          "self":"http://192.168.2.15:3000/api/v1/manager/clients/918ba9c6-bde4-543a-a280-afbc1bf698ad/relationships/client-telephones", 
+          "related":"http://192.168.2.15:3000/api/v1/manager/clients/918ba9c6-bde4-543a-a280-afbc1bf698ad/client-telephones"
+        }
+      }
+    }
+  }
+}
+```
+
+### List
+
+You also can list your resources using a command line or accessing a route, both cases are described below. In this exeample we use the manager.
+
```

Comment:
Correct: "example" not exeample:  You also can list your resources using a command line or accessing a route, both cases are described below. In this example we use the manager.